bn254-fr/src/lib.rs

```diff
@@ -293,10 +293,29 @@ impl Div for Bn254Fr {
     }
 }
 
-impl Distribution<Bn254Fr> for Standard {
+impl Distribution<Bn254Fr> for StandardUniform {
     #[inline]
     fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> Bn254Fr {
```

Comment:
This implementation looks correct, but worth noting that the [old version](https://github.com/privacy-scaling-explorations/halo2curves/blob/3bfa6562f0ddcbac941091ba3c7c9b6c322efac1/derive/src/field/mod.rs#L383) uses a different implementation, which is (plausibly?) constant time at the cost of a slight bias.

I think this is fine, as we haven't been aiming for constant time (in the sense of security against side channel attacks), and I don't believe any recursion code would need to reproduce this behavior since rng APIs aren't normally used in our Fiat-Shamir implementations. cc @jtguibas @jonathanpwang, correct me if I'm wrong